src/js/api/brand.js

```diff
@@ -491,6 +491,96 @@ ripe.Ripe.prototype.onInitialsP = function(options) {
     });
 };
 
+/**
+ * Returns the result of the server side validation of model configurations
+ * considering a given build's brand, model and version.
+ *
+ * @param {Object} options An object with options, such as:

```

Comment:
Some stuff missing:

```
        version = self.field("version", None)
        variant = self.field("variant", None)
        parts = self.field("p", [], cast = list)
        initials = self.field("initials", None)
        engraving = self.field("engraving", None)
        initials_extra = self.field("initials_extra", None, multiple = True)
        gender = self.field("gender", None)
        size = self.field("size", None)
```

Reply:
at least gender and variant

Reply:
I believe everything is here
gender: https://github.com/ripe-tech/ripe-sdk/blob/316b55f4ba8d49a422470a3db660e0f3c2cd72d5/src/js/api/brand.js#L547
variant: https://github.com/ripe-tech/ripe-sdk/blob/316b55f4ba8d49a422470a3db660e0f3c2cd72d5/src/js/api/brand.js#L541

They are presented in the same order as in ripe-core

Reply:
Yep, they're just missing in the javadoc.